lib/task/tools/runCustomLogicTask.class.php

```diff
@@ -40,6 +40,7 @@ class runCustomLogicTask extends sfBaseTask
   protected function configure()
   {
     $this->addOptions(array(
+      new sfCommandOption('log', null, sfCommandOption::PARAMETER_NONE, 'Log execution of PHP file'),
```

Comment:
Should we allow users to optionally specify an alternate log file name (with default of tools_run.log) in case they were testing something a few times?  Otherwise it will blow the log away with each run unless they rename it or copy it to a new location.  Do you think this would be useful/necessary?